smac/utils/configspace.py

```diff
@@ -52,11 +52,22 @@ def get_types(
     -------
     The bounds for the instance features are *not* added in this function.
     """
+    # NOTE (eddiebergman): It would be better return a dict[str, tuple[int, float, float]]
+    #   where the str is the hyperparameters name and the dictionary is in sorted order by
+    #   the hyperparameters name.
+    #
+    # We are currently relying on the order of the hyperparameters of the space always
```

Comment:
In that case, can we ensure that the orders of the hp bounds are always correct (fit the order of confispace)?